locales/en/app.ftl

```diff
@@ -934,3 +934,9 @@ find-out-if-2 = Find out if you were involved in this breach
 find-out-if-description = We’ll help you quickly see if your email address was exposed in this breach, and understand what to do next.
 
 breach-detail-cta-signup = Check for breaches
+
+## Floating banner
+
+floating-banner-text = Boost your online security with news, tips, and updates from Mozilla.
```

Comment:
Q: Should this use `-brand-mozilla`?

Reply:
Ah, CI logger error says yes.

```sh
Run moz-fluent-lint ./locales/en --config .github/linter_config.yml
Files to analyze: 10.

            File path: app.ftl
            Message ID: floating-banner-text
            Position: line 940 column 24
            Error (CO01): Strings should use the corresponding terms instead of hard-coded brand names (Mozilla)
Error: Process completed with exit code 1.
```